src/app/app-routing.module.ts

```diff
@@ -21,9 +22,11 @@ const routes: Routes = [
     canActivate: [AuthGuard, PassengerGuard],
   },
   {
-    path: 'dashboard',
-    component: AdminPanelComponent,
-    canActivate: [AuthGuard, AdminGuard],
+    path: '',
```

Comment:
can we use canLoad guard?

Reply:
As discussed verbally, we can consider this for further enhancement. Thank you.